data/json/vehicleparts/vehicle_parts.json

```diff
@@ -639,7 +639,7 @@
     "looks_like": "minifridge",
     "damage_modifier": 80,
     "durability": 100,
-    "epower": "-45 W",
+    "epower": "-12 W",
```

Comment:
```suggestion
    "epower": "-36 W",
```

Reply:
While 35W is enough good for 450L freezer, mini variant cant techically consume more power
As variant https://www.osculati.com/en/11211-50.042.00/isotherm-refrigeratorfreezer-bi75-75-l
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/102726167/76378b2e-eff5-492c-8ff6-a23559847fd5)
350w/24 = 14.5W
So make at least 2 times lesser, than its big variant. 17W is quite ok even for hand made fridge